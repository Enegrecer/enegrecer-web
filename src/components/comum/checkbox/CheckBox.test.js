import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import CheckBox from './../checkbox';

describe('Checkbox', () => {
  it('deve possuir estrutura definida ', () => {
    const onClick = jest.fn();
    const checkBox = mount(<CheckBox
      id={'souAVitima'}
      label={'Sou a VÃ­tima'}
      onClick={onClick}
    />);
    const checkBoxTextoJson = toJson(checkBox);
    expect(checkBoxTextoJson).toMatchSnapshot();
  });
});

