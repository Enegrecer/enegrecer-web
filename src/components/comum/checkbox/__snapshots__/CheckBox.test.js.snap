// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox deve possuir estrutura definida  1`] = `
<checkBox
  id="souAVitima"
  label="Sou a Vítima"
  onClick={[Function]}
>
  <p>
    <input
      id="souAVitima"
      type="checkbox"
    />
    <label
      htmlFor="souAVitima"
    >
      Sou a Vítima
    </label>
  </p>
</checkBox>
`;
